lighthouse-core/report/html/report-styles.css

```diff
@@ -854,7 +857,17 @@ summary.lh-passed-audits-summary {
   text-align: left;
   min-width: 250px;
   white-space: nowrap;
-  max-width: var(--url-col-max-width);
+  max-width: 0;
+}
+
+/* Keep bytes columns narrow if they follow urls */
+.lh-table-column--url + th.lh-table-column--bytes,
+.lh-table-column--url + .lh-table-column--bytes + th.lh-table-column--bytes {
+  min-width: var(--bytes-col-width);
+  width: var(--bytes-col-width);
+}
+th.lh-table-column--bytes {
+  white-space: normal!important;
```

Comment:
are we overwriting anything anymore? do we still need to do this?

Reply:
good call thx